content/components/wifi.md

```diff
@@ -327,7 +327,7 @@ on_...:
 ```
 
 > [!NOTE]
-> Be aware that if you disable WiFi, the API timeout will need to be disabled otherwise the device will reboot.
+> If you disable WiFi, you must enable WiFi again before the API timeout is reached or the device will reboot. You can disable the API timeout to avoid this limitation.
```

Comment:
```suggestion
> Be mindful of the reboot timeouts set for both the API component and WiFi component if you disable WiFi. If WiFi
> remains off for longer than the duration of either timeout, the device will reboot!
```

...and we should make "API component" and "WiFi component" links to their relevant sections